Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -2642,6 +2642,9 @@ bool RenderLayerCompositor::requiresCompositingLayer(const RenderLayer& layer, R
         return false;
     }
 
+    if (isHiddenSubframe(renderer))
```

Comment:
This is the wrong place for this; it should be in `requiresCompositingForFrame()`.

Reply:
The idea to place it here is to ignore any other reasons for composting when dealing with hidden subframes, like requiresCompositingForTransform which would be true in the tests and would override requiresCompositingForFrame.